src/FitsViewerExt.js

```diff
@@ -178,13 +177,14 @@ export class FitsViewerWidget extends Widget {
 }
 
 
-function tellLabToloadFileToServer( filename, firefly) {
-    const {fetchUrl}= firefly.util;
-    const UL_URL= window.document.location.href+ `/sendToFirefly?path=${filename}`;
-    const options = { method: 'GET' };
-    return fetchUrl(UL_URL,options,false, false)
-        .catch( e => {
-            console.log('Got Error from upload request');
+function tellLabToLoadFileToServer( path, firefly) {
+    const {searchParams,origin,pathname}= new URL(window.document.location.href);
+    searchParams.append('path',path);
+    const slashMaybe= pathname.endsWith('/') ? '' : '/';
+    const newURL= `${origin}${pathname}${slashMaybe}sendToFirefly?${searchParams.toString()}`;
```

Comment:
Is there a standard way to join parts of a url? I didn't find one in a quick search. I saw a `url-join` package that is installable via npm.

Reply:
I don't want to bring in extra npm packages. For just this. I am trying to stick more this standard JS. By use `origin` and searchParams the hard part is handled. The funny thing is that `URL` constructor should have been able to handle adding to the path if it ended with `/` but the `?` confuses it as well. I would actually have to do more parsing the make that work. I think this approach is the safest.